pkg/cmd/pulumi/crypto_local.go

```diff
@@ -35,8 +37,27 @@ func readPassphrase(prompt string) (string, error) {
 	if phrase, ok := os.LookupEnv("PULUMI_CONFIG_PASSPHRASE"); ok {
 		return phrase, nil
 	}
+	if phraseFile, ok := os.LookupEnv("PULUMI_CONFIG_PASSPHRASE_FILE"); ok {
+		phraseFilePath, err := filepath.Abs(phraseFile)
+		if err != nil {
+			return "", errors.New("unable to construct a path the PULUMI_CONFIG_PASSPHRASE_FILE")
```

Comment:
```suggestion
			return "", errors.Wrap(err, "unable to construct a path the PULUMI_CONFIG_PASSPHRASE_FILE")
```

(Suggestion applies to all errors)

Reply:
This requires changing the import to `"github.com/pkg/errors"`